src/tike/operators/cupy/ptycho.py

```diff
@@ -103,18 +103,18 @@ def fwd(self, probe, scan, psi, **kwargs):
             self.diffraction.fwd(
                 psi=psi,
                 scan=scan,
-                probe=probe,
+                probe=probe[..., 0, 0, :, :],
```

Comment:
Does this change mean you use only the first probe in a multi-probe set? Is there any shape change that makes you use ... to replace ntheta?

Reply:
If you look at the lines above, the docstring for the `Ptycho` Operator specifies that we were *already* using only the first probe in a multi-probe set. What has changed in this PR that the `Convolution` operator expects a probe of a different shape, so this line strips two of the dimensions from the probe. In the next PR, I will actually remove this constraint and allow multiple probes to be processed simultaneously. However, this ability requires a new feature in the `Patch` operator.

This shape change is unrelated to the value of `ntheta`. I have replaced `ntheta` with `...` in the docs because the Operator now functions with arbitrary leading dimensions.